backend/src/main/resources/application-local.properties

```diff
@@ -0,0 +1,3 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/messenger
```

Comment:
It seems to me pushing files with local settings, it isn't good approach. Please remove it

Reply:
Added the file to .gitignore and removed from index (git rm --cached name_of_file)